mPowerSDK/Onboarding/APHOnboardingManager.h

```diff
@@ -38,4 +38,9 @@
 - (ORKTaskViewController *)instantiateOnboardingTaskViewController:(BOOL)signUp;
 - (ORKTaskViewController *)instantiateConsentViewController;
 
+// @protected
```

Comment:
I use that as a marker that the code below is intended as protected methods (because there isn't actually a syntax for that in obj-c).